projects/ag-grid/documentation/code-examples/data-grid/data-manager-added/data-manager-data-grid-docs-demo.component.ts

```diff
@@ -173,6 +174,7 @@ export class SkyDataManagerDataGridDemoComponent implements OnInit {
     private changeDetector: ChangeDetectorRef,
     private dataManagerService: SkyDataManagerService
   ) {
+    this.noRowsTemplate = `No results found`;
```

Comment:
```suggestion
    this.noRowsTemplate = `No results found.`;
```

Reply:
fixed in commit [56545e6 - "add periods."](https://github.com/blackbaud/skyux-ag-grid/pull/193/commits/56545e6d112a5dc513982f5560e10cdc4a119b42)